tests/mocha/event_test.js

```diff
@@ -223,15 +223,17 @@ suite('Events', function() {
 
     suite('With variable getter blocks', function() {
       setup(function() {
-        this.genUidStub = createGenUidStubWithReturns(this.TEST_BLOCK_ID);
-        this.block =
-            createSimpleTestBlock(this.workspace, 'field_variable_test_block');
+        this.genUidStub = createGenUidStubWithReturns(
+            [this.TEST_BLOCK_ID, 'test_var_id', 'test_group_id']);
+        // Disabling events when creating a block with variable can cause issues
+        // at workspace dispose.
+        this.block = new Blockly.Block(
```

Comment:
Would it make more sense to instead just pass an optional parameter (to not disable events for `createSimpleTestBlock`). In other words, is there value in keeping these tests consistent in how they're creating the Block (by going through `createSimpleTestBlock`), and for the cases where you don't want to disable events during creation, that can be accommodated through an optional parameter.

Reply:
I prefer not to add a flag to `createSimpleTestBlock` because I don't think the naming makes sense to use `createSimpleTestBlock` to create a block other than `simple_test_block`. I could rename, but this is the only case that's different, so it makes more sense to me to not re-purpose the helper.